pkg/pow/pool.go

```diff
@@ -136,7 +136,7 @@ func (p *StratumPool) GetWork(input chan *Work) (*Work, bool) {
 					Difficulty: miningNotify.MedianDifficulty,
 					// Difficulty: big.NewInt(10000000),
 					// Difficulty: big.NewInt(6377077812),
-					RequestID: big.NewInt(1),
+					RequestIDs: [5]*big.Int{big.NewInt(1)},
```

Comment:
I don't understand this bit.

What is a Stratum pool?

Reply:
I think I was just setting a base value and we set them later on....but not sure